app/controllers/articles_controller.rb

```diff
@@ -285,7 +285,7 @@ def handle_notifications(updated)
       Notification.remove_all_by_action_without_delay(notifiable_ids: @article.id, notifiable_type: "Article",
                                                       action: "Published")
       if @article.comments.exists?
-        Notification.remove_all(notifiable_ids: @article.comments.pluck(:id),
+        Notification.remove_all(notifiable_ids: @article.comments.ids,
```

Comment:
`pluck(:id)` is literally the same as `ids` which is a builtin method in Rails

https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-ids